src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/management/BannerTestsIT.java

```diff
@@ -0,0 +1,150 @@
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("title", is("Banner 1"))
+      .body("contents", containsString("Banner 1 contains text"))
+      .body("link", nullValue())
+      .body("textColor", nullValue())
+      .body("backgroundColor", nullValue());
+  } 
+  
+  @Test
+  public void testListBanners() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/banners", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(3))
+      .body("id[2]", notNullValue())
+      .body("title[2]", is("Banner 3 &#8211; the full featured one"))
+      .body("contents[2]", is(""))
+      .body("link[2]", is("http://link.example.com"))
+      .body("textColor[2]", is("#e03339"))
+      .body("backgroundColor[2]", is("rgba(221,153,51,0.3)"));
+  } 
+  
+  @Test
+  public void testBannersNoTitle() {
+    String organizationId = getOrganizationId(0);
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/banners/{bannerId}", organizationId, getOrganizationBannerId(organizationId, 1))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("title", nullValue())
+      .body("contents", is(""))
+      .body("link", nullValue())
+      .body("textColor", nullValue())
+      .body("backgroundColor", nullValue());
+  } 
+  
+  @Test
+  public void testOrganizationBannersNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationAnnouncecmentId = getOrganizationBannerId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/banners/%s", organizationId, organizationAnnouncecmentId));
+    assertEquals(3, countApiList(String.format("/organizations/%s/banners", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/banners/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/banners/%s", incorrectOrganizationId, organizationAnnouncecmentId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/banners", incorrectOrganizationId)));
+  }
+
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+    
+  private void createManagementSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createManagementSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)